internal/pkg/helm/convert.go

```diff
@@ -375,3 +378,73 @@ func InstallAnsibleTasks(roleDirectory string) {
 			destinationTasksYamlFile)
 	}
 }
+
+<<<<<<< HEAD
+// Determines whether a YAML variable (i.e., "image.pullPolicy") refers to a boolean within a given Helm Chart context.
+// TODO: This is currently unused, as the testing method skips client use.  In the future, call this from cmd.go.
+func IsBooleanYamlsValue(chartClient HelmChartClient, variableName string) (bool, error){
+	variableArray := strings.Split(variableName, goTemplateMemberAccessOperator)
+
+	raw, err := chartutil.ReadValues([]byte(chartClient.Chart.Values.Raw))
+	if err != nil {
+		return false, errors.New("Could not parse " + variableName)
+	}
+
+	// Determine depth and make sure that the path that was passed has at least one element.
+	variableDepth := len(variableArray)
+	if variableDepth < 1 {
+		return false, errors.New("Cannot split " + variableName + " appropriately")
+	}
+
+	return IsBooleanYamlValue(raw, variableArray)
```

Comment:
The func name looks similar to this function as plural, which might be hard to read.

Reply:
I removed this function for now.